mongo/src/storages/mongo/cursor.cpp

```diff
@@ -15,6 +15,10 @@ Cursor& Cursor::operator=(Cursor&&) noexcept = default;
 
 bool Cursor::HasMore() const { return impl_->IsValid(); }
 
+uint32_t Cursor::GetBatchSize() const { return impl_->GetBatchSize(); }
+
+void Cursor::SetBatchSize(uint32_t size) { impl_->SetBatchSize(size); }
```

Comment:
Idea: Changing signature to `Cursor &` and `return *this` makes possible to write chaining methods like:  
```
auto cursor = collection.Find(filter).SetBatchSize(N);
```
It is similar to mongo [example](https://www.mongodb.com/docs/manual/reference/method/cursor.batchSize/#example) syntax